docs/source/getting-started/model-registry/step1-register-model.rst

```diff
@@ -0,0 +1,63 @@
+Register a Model
+=================
+
+To effectively use the Model Registry, it's necessary to first install and configure MLflow. In this
+quick start guide, we'll use a local tracking server which saves information about your MLflow 
+resources in a local directory. More specifically, models will be located in the `./mlruns/models` 
+directory.
+
+.. note::
+    For production use cases, we advise using a remote server. In this guide, we're using a local 
+    server for demonstration purposes.
+
+
+Step 1: Register a Model
+--------------------------------
+
+With the MLflow library installed, we can now create and register a model. Below we leverage 
+`sklearn`'s `RandomForestRegressor` fit on a generated dataset. This model will be logged
+to a local model registry at `./mlruns/models`. 
+
+.. code-section::
+    .. code-block:: python 
+        :name: create-model 
+
+        from sklearn.datasets import make_regression
+        from sklearn.ensemble import RandomForestRegressor
+        from sklearn.metrics import mean_squared_error
+        from sklearn.model_selection import train_test_split
+
+        import mlflow
+        import mlflow.sklearn
+        from mlflow.models import infer_signature
+
+        with mlflow.start_run() as run:
+            X, y = make_regression(n_features=4, n_informative=2, random_state=0, shuffle=False)
+            X_train, X_test, y_train, y_test = train_test_split(
+                X, y, test_size=0.2, random_state=42
+            )
+            params = {"max_depth": 2, "random_state": 42}
+            model = RandomForestRegressor(**params)
+            model.fit(X_train, y_train)
+
+            # Infer the model signature
+            y_pred = model.predict(X_test)
+            signature = infer_signature(X_test, y_pred)
```

Comment:
@BenWilson2 for the model signature, I'm wondering whether we should have the signature inferred from input example instead. This would supply both the input example and model signature. When we infer signatures from input example, we also make sure to score the input example with the pyfunc wrapped model, which is more accurate.

Reply:
cc: @serena-ruan let me know if this makes sense for input params too.

Reply:
This code snippet is copied from the current model registry docs - [exmaples](https://github.com/search?q=repo%3Amlflow%2Fmlflow%20X%2C%20y%20%3D%20make_regression(n_features%3D4%2C%20n_informative%3D2%2C%20random_state%3D0%2C%20shuffle%3DFalse)&type=code). Given this is a model registry tutorial, does signature inference matter? If so, I suggest we merge the PR then change all instance of the signature inference (which I can do).